src/miner_keys.erl

```diff
@@ -94,8 +110,18 @@ print_keys(_) ->
     MaybeB58 = fun(undefined) -> undefined;
                   (Key) -> libp2p_crypto:pubkey_to_b58(Key)
                end,
+    MaybeUUIDv4 = fun(undefined) -> undefined;
+                  (Key) ->
+                      %% only production blackspots read onboarding_key from a file
+                      case filelib:is_file("/mnt/uboot/onboarding_key") of
```

Comment:
Don't use the file, but catch the b58 encoding error and do the binary_to_list option?

Reply:
And is MabeB58 dead code now?

Reply:
No, I believe we still need it for Helium Hotspots that read the `onboarding_key` out of the ECC608.

Reply:
> Don't use the file, but catch the b58 encoding error and do the binary_to_list option?

Exceptions scare me. I actually encountered that b58 encoding error while implementing this function. I prefer this way.